lib/test/t_PointToPointConnection_std.cxx

```
@@ -0,0 +1,95 @@
+ *  This library is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU Lesser General Public License as published by
+ *  the Free Software Foundation, either version 3 of the License, or
+ *  (at your option) any later version.
+ *
+ *  This library is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU Lesser General Public License for more details.
+ *
+ *  You should have received a copy of the GNU Lesser General Public
+ *  along with this library.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+#include "openturns/OT.hxx"
+#include "openturns/OTtestcode.hxx"
+
+using namespace OT;
+using namespace OT::Test;
+
+int main(int argc, char *argv[])
+{
+  TESTPREAMBLE;
+  OStream fullprint(std::cout);
+
+  try
+  {
+    // Construction based on two functions
+    {
+      /* Create an instance */
+      PointToPointConnection myFunc(SymbolicFunction("x", "sin(x)"), SymbolicFunction("x", "cos(x)"));
+
+      fullprint << "myFunc=" << myFunc << std::endl;
+      /* Get the input and output description */
+      fullprint << "myFunc input description=" << myFunc.getInputDescription() << std::endl;
+      fullprint << "myFunc output description=" << myFunc.getOutputDescription() << std::endl;
+      /* Get the input and output dimension */
+      fullprint << "myFunc input dimension=" << myFunc.getInputDimension() << std::endl;
+      fullprint << "myFunc output dimension=" << myFunc.getOutputDimension() << std::endl;
+      /* Connection on a point */
+      Point point(myFunc.getInputDimension(), 1.0);
+      fullprint << "point=" << point << std::endl;
+      fullprint << "myFunc(point)=" << myFunc(point) << std::endl;
+      fullprint << "myFunc.gradient(point)=" << myFunc.gradient(point) << std::endl;
+      fullprint << "myFunc.hessian(point)=" << myFunc.hessian(point) << std::endl;
```

Test fails here on my box ; I get a 1e-07 instead of a 0 when printing ```myFunc.hessian(point)```.